src/vars.d.ts

```diff
@@ -22,6 +22,7 @@
 declare namespace NodeJS {
   interface Global {
     API_ENDPOINT: string
+    API_SECRET: string
```

Comment:
There is already `secrets.d.ts` for that

Reply:
Can we merge all variable declarations in one file?

Reply:
I'd prefer to keep them split because they are handled differently by Cloudflare.
- Secrets either need to be added via CLI or manually via Dashboard
- Variables are defined in wrangler.toml
- KV need to be added first and then registered in wrangler.toml

When we split those three files, we know immediately what to do when one of the file changes.